docs/excel/custom-functions-naming.md

```diff
@@ -10,7 +10,9 @@ A custom function is identified by an **id** and **name** property in the JSON m
 
 Function names and function IDs share some common requirements:
 
-- They must only use alphanumeric characters (including Unicode), the numbers zero through nine, underscores, and periods.
+- Function ids may only use characters A through Z, numbers zero through nine, underscores, and periods.
+
+- Function names may use any Unicode alphanumeric character, underscore, and period.
```

Comment:
"alphanumeric" has a certain meaning in Unicode which is larger than we support for custom functions. Because numeric is not just digits 0 to 9 but can include other numeric forms.

It is more accurate to say "any Unicode **Alphabetic** character" because that is a well-defined character property. https://www.unicode.org/reports/tr44/#Alphabetic

https://www.unicode.org/versions/Unicode12.0.0/ch04.pdf
Section 4.5 describes General Category for Unicode characters. 
Section 4.6 describes Numeric Values.
Section 4.10 describes Alphabetic.
https://www.unicode.org/reports/tr44/tr44-24.html#Alphabetic shows how the Alphabetic character set is derived.
The actual characters for Alphabetic are found here in [DerivedCoreProperties.txt](https://www.unicode.org/Public/UCD/latest/ucd/DerivedCoreProperties.txt) in the section containing `# Derived Property: Alphabetic`

So just to be clear, the function names can use an Unicode "Alphabetic" character, the digits 0 through 9, underscore, and period.  It is misleading to say "alphanumeric" and should specify only "digits 0 through 9" to be accurate.

Reply:
Ah ok! That makes total sense. Thanks!